dynamic.tex

```diff
@@ -426,6 +426,23 @@ \section{Multiple Files}\label{s:dynamic-multiple}
 </html>
 \end{minted}
 
+
+\begin{aside}{Installing React and ReactDOM}
+
+Now that we've made a project folder using \texttt{npm init},
+and we have a \texttt{project.json} file,
```

Comment:
"which created a project.json file" maybe, to make it clear that `npm init` does the creating?